apps/dashboard/src/utils/i18nUtils.ts

```diff
@@ -9,28 +27,65 @@ const i18n = createI18n({
     legacy: false,
     globalInjection: true,
     messages: {
```

Comment:
Why are the `components` and `modules` fields present in the imported json files?

I think it my be clearer if you omit those fields from the imported json files (and thus move all fields up a level), and instead import the files as, for example, `footer_en_components`, and destruct those json files directly.

Reply:
I don't know exactly what you mean? If you mean the keys in the json files themselves, that is so the lintr recognises everything. Other setups don't work, I've tried.